packages/loot-core/src/server/accounts/rules.js

```diff
@@ -416,9 +444,22 @@ export class Action {
     this.options = options;
   }
 
-  exec(object) {
+  exec(object, transaction) {
```

Comment:
I was also wondering if it is worth adding an assert that transaction is not empty or just the id. I think there may be a few places to track down where not all of the information is added yet.